ibis/backends/dask/execution/util.py

```diff
@@ -37,3 +39,24 @@ def make_selected_obj(gs: SeriesGroupBy):
         return gs.obj.set_index(gs.index, drop=False)[
             gs._meta._selected_obj.name
         ]
+
+
+def maybe_wrap_scalar(result: Any, expr: ir.Expr) -> Any:
```

Comment:
can we call coerce_to_output then

Reply:
I'm not convinced yet that we'll need it (or that dask udfs are going to operate like pandas udfs) so I picked I name that reflected the existing functionality.